e2e/connect/acls.go

```diff
@@ -250,6 +265,47 @@ func (tc *ConnectACLsE2ETest) TestConnectACLsConnectDemo(f *framework.F) {
 	t.Log("connect legacy job with ACLs enable finished")
 }
 
+func (tc *ConnectACLsE2ETest) TestConnectACLsConnectDemoNamespaced(f *framework.F) {
+	t := f.T()
+
+	t.Log("test register Connect job w/ ACLs enabled w/ operator token")
+
+	// === Setup ACL policy within a namespace and mint Operator token ===
+
+	// create a namespace
+	namespace := tc.createConsulNamespace("ns-"+uuid.Short(), f)
+	t.Log("created namespace:", namespace)
```

Comment:
Assuming these run against HCP based on the commit history, and therefore this is supported?

Reply:
Correct, our e2e runs against HCP Consul.